src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/LegacyReleaseController.cs

```diff
@@ -18,16 +20,24 @@ public LegacyReleaseController(ILegacyReleaseService legacyReleaseService)
         {
             _legacyReleaseService = legacyReleaseService;
         }
-        
-        [HttpGet("api/legacy-releases/{id}")]
+
+        [HttpGet("publications/{publicationId:guid}/legacy-releases")]
+        public async Task<ActionResult<List<LegacyReleaseViewModel>>> GetLegacyReleases(Guid publicationId)
```

Comment:
Not a biggy, but could name this `ListLegacyReleases`. Same with the service method as well.

Reply:
Thanks, made this change now. 🙂